test/auro-pane.test.js

```diff
@@ -52,12 +52,21 @@ describe('auro-pane', () => {
     expect(isPanePricedClassPresent(el)).to.be.true;
   })
 
+  it('displays two dashes when price attribute is empty', async () => {
+    const el = await fixture(html`
+      <auro-pane date="2020-09-09" price=""></auro-pane>
+    `);
+
+    expect(getButtonText(el)).to.equal("Wed Sep 9 --");
+    expect(isPanePricedClassPresent(el)).to.be.true;
+  })
+
   it('allows empty price when disabled', async () => {
     const el = await fixture(html`
       <auro-pane date="2020-09-09" price="" disabled></auro-pane>
     `);
 
-    expect(getButtonText(el)).to.equal("Wed Sep 9");
+    expect(getButtonText(el)).to.equal("Wed Sep 9 --");
     expect(isPanePricedClassPresent(el)).to.be.true;
   })
```

Comment:
There is no functional code for disabled. Just code that passes the makes sure the button element reflects the value. I'm not sure what value this test adds considering the new test added on line 55.

Thoughts?

Reply:
You are correct, this test case does not reflect the title of the test and is a repeat of the test on line 55. I will edit this test to reflect a valuable test case.